### DISCUSSION TITLE
Cant access Rawgithubusercontent.com

### DISCUSSION THREAD

**Discussion by DMNxD4rk** [community/community#42518] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can't access the website rawgithubusercontent.com i contacted my isp and they said the issue is from the website i can access it with vpn tho idk what's the issue i am indian and use JIO isp

**Comment by jfullstackdev** [comment]
as far as I know, GitHub does not limit that, it's usually the ISP really or maybe you are in a place that GitHub is not allowed
